server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java

```diff
@@ -0,0 +1,216 @@
+        log.trace("table: {}, adding: {}", tableName, update.prettyPrint());
+        batchWriter.addMutation(update);
+
+        Mutation delete = new Mutation(key.getRow());
+        delete.at().family(DataFileColumnFamily.STR_NAME).qualifier(file).delete();
+        log.trace("table {}: deleting: {}", tableName, delete.prettyPrint());
+        batchWriter.addMutation(delete);
+      } catch (MutationsRejectedException ex) {
+        // include constraint violation info in log - but stop upgrade
+        log.warn(
+            "Failed to update file reference for table: " + tableName + ". row: " + key.getRow());
+        log.warn("Constraint violations: {}", ex.getConstraintViolationSummaries());
+        throw new IllegalStateException("File conversion failed. Aborting upgrade", ex);
+      }
+    }
+  }
+
+  @VisibleForTesting
+  void removeChoppedCF(final Key key, final BatchWriter batchWriter, final String tableName) {
+    Mutation delete = null;
+    try {
+      delete = new Mutation(key.getRow()).at().family(ChoppedColumnFamily.STR_NAME)
+          .qualifier(ChoppedColumnFamily.STR_NAME).delete();
+      log.warn(
+          "Deleting chopped reference from:{}. Previous split or delete may not have completed cleanly. Ref: {}",
+          tableName, delete.prettyPrint());
+      batchWriter.addMutation(delete);
+    } catch (MutationsRejectedException ex) {
+      log.warn("Failed to delete obsolete chopped CF reference for table: " + tableName + ". Ref: "
+          + delete.prettyPrint() + ". Will try to continue. Ref may need to be manually removed");
+      log.warn("Constraint violations: {}", ex.getConstraintViolationSummaries());
+      throw new IllegalStateException(
+          "Failed to delete obsolete chopped CF reference for table: " + tableName, ex);
+    }
+  }
+
+  @VisibleForTesting
+  void removeExternalCompactionCF(final Key key, final BatchWriter batchWriter,
+      final String tableName) {
+    Mutation delete = null;
+    try {
+      delete = new Mutation(key.getRow()).at().family(ExternalCompactionColumnFamily.NAME)
+          .qualifier(key.getColumnQualifier()).delete();
+      log.debug(
+          "Deleting external compaction reference from:{}. Previous compaction may not have completed. Ref: {}",
+          tableName, delete.prettyPrint());
+      batchWriter.addMutation(delete);
+    } catch (MutationsRejectedException ex) {
+      log.warn("Failed to delete obsolete external compaction CF reference for table: " + tableName
+          + ". Ref: " + delete.prettyPrint()
+          + ". Will try to continue. Ref may need to be manually removed");
+      log.warn("Constraint violations: {}", ex.getConstraintViolationSummaries());
+      throw new IllegalStateException(
+          "Failed to delete obsolete external compaction CF reference for table: " + tableName, ex);
+    }
+  }
+
+  /**
+   * Quick sanity check to see if value has been converted by checking the candidate can be parsed
```

Comment:
Could use language that is more inclusive here.

```suggestion
   * Quick validation to see if value has been converted by checking the candidate can be parsed
```

Reply:
Fixed in cc6b6e25801c